CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 Changelog
 =========
+# 0.47.10 / 2023-08-07
+
+* [IMPROVEMENT] Improvements in how JMXFetch handles communicating back to the Agent. The TLS of the HTTP client used can now be configured, extra logging has been added around the SSL Context and 'TLS' as min protocol version used in the `dummyTrustManager` [#436][] 
```

Comment:
Maybe worth documenting the system property than can be used to override this. https://github.com/DataDog/jmxfetch/pull/436/files#diff-7b667f02364f0f4390a5f2db300223a30859c670c4a9515f94f40a2cad8fa4b6R29

`-Djmxfetch.min_tls_version=TLS` is the default, but can be overridden if needed. Legacy value is `SSL`, so if there are unexpected issues related to TLS, this system property can be used to override and go back to previous behavior.